lib/kms.js

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const {keystores} = require('bedrock-kms');
+const brHttpsAgent = require('bedrock-https-agent');
+const {AsymmetricKey, Hmac, Kek, KeystoreAgent, KeyAgreementKey, KmsClient} =
+  require('webkms-client');
+
+// load config defaults
+require('./config');
+
+// module API
+const api = {};
+module.exports = api;
+
+api.createKeystore = async ({capabilityAgent, referenceId} = {}) => {
+  const kmsBaseUrl = `${bedrock.config.server.baseUri}/kms`;
+  // create keystore
+  const config = {
+    sequence: 0,
+    controller: capabilityAgent.id,
+    // TODO: add `invoker` and `delegator` using arrays including
+    // capabilityAgent.id *and* identifier for backup key recovery entity
+    invoker: capabilityAgent.id,
+    delegator: capabilityAgent.id
```

Comment:
Let's remove `invoker` and `delegator` here and the TODO. We want to do capability based "key recovery".

Reply:
Punting on removing invoker and delegator due to time constraints.